frontend/src/metabase/ui/components/overlays/Popover/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Popover } from "@mantine/core";
+import type { PopoverDropdownProps } from "@mantine/core";
+
+export type { PopoverBaseProps, PopoverProps } from "@mantine/core";
+export { getPopoverOverrides } from "./Popover.styled";
+
+const MantinePopoverDropdown = Popover.Dropdown;
+
+const PopoverDropdown = Object.assign(function PopoverDropdown(
+  props: PopoverDropdownProps,
+) {
+  return <MantinePopoverDropdown {...props} data-popover="mantine-popover" />;
```

Comment:
This little `data-popover` is a way for me to target Mantine popover in `mantinePopover()` E2E util.